draft-ietf-quic-multipath.md

```diff
@@ -448,12 +448,19 @@ PATH_CHALLENGE arrives in a packet using a non-zero length Connection ID.
 ### Effect of RETIRE_CONNECTION_ID Frame {#retire-cid-close}
 
 Receiving a RETIRE_CONNECTION_ID frame causes the endpoint to discard
-the resources associated with that connection ID. If the connection ID
+the resources associated with that Connection ID. If the connection ID
 was used by the peer to identify a path from the peer to this endpoint,
 the resources include the list of received packets used to send
-acknowledgements. The peer MAY decide to keep sending data using
+acknowledgements. The peer can keep sending data using
 the same IP addresses and UDP ports previously associated with
 the connection ID, but MUST use a different connection ID when doing so.
+E.g this can happen if the Connection ID issuer requests retirement of a
+Connection ID using the Retire Prior To field in the NEW_CONNECTION_ID Frame.
+If no new connection ID is available anymore, the endpoint cannot send on
+this path anymore but has to wait for the idle time-out before closing
```

Comment:
I'm not sure that the endpoint "has to wait" the idle timeout to close the path. I could also send a PATH_ABANDON to explicitly close the path, right? Instead, do we want to state here that an inactivity larger than the idle timeout implies an implicit path closure?

Reply:
Currently it can't send an PATH_ABANDON because if we don't have a CID we don't have a valid path ID.

Reply:
Oh indeed, I went too quick on this one :-)

Reply:
I do mention this issue in #169

Reply:
Actually we could say that the endpoint should first send the path abandoned frame and then the retire CID frame. However, those frames probably don't have to be processed in order and we need to address issue #137 properly. Or alternative you have to wait until the path abandoned frame is ack'ed and then send the retire CID frame but that would introduce a delay. Not sure how bad that is.